common/src/nexus_config.rs

```diff
@@ -115,8 +130,9 @@ pub struct DeploymentConfig {
     pub dropshot_external: ConfigDropshotWithTls,
     /// Dropshot configuration for internal API server.
     pub dropshot_internal: ConfigDropshot,
-    /// Portion of the IP space to be managed by the Rack.
-    pub subnet: Ipv6Subnet<RACK_PREFIX>,
```

Comment:
This was only used for finding DNS servers. We change it to be more appropriately named, and to provide a variant for access-by-address, for tests.